docs/Using Any API/large-responses.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Large Responses"
+permalink: "docs/large-responses/"
+whatsnext: {"Make an Existing Job Request":"/docs/existing-job-request/", "API Reference":"/docs/chainlink-framework/", "Contract Addresses":"/docs/decentralized-oracles-ethereum-mainnet/"}
+metadata: 
+  title: "Large Responses"
+  description: "Learn how to make a GET request to an API and receive a dynamically sized large response from a smart contract, using Chainlink.."
+  image: 
+    0: "/files/930cbb7-link.png"
+---
+This page explains how to make an HTTP GET request to an external API from a smart contract, using Chainlink's [Request & Receive Data](../request-and-receive-data/) cycle and then receive large responses.
+
+# Large Response
+
+To consume an API with a large responses, your contract should inherit from <a href="https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/ChainlinkClient.sol" target="_blank">`ChainlinkClient`</a>. This contract exposes a struct called `Chainlink.Request`, which your contract should use to build the API request. The request should include the oracle address, the job id, the fee, adapter parameters, and the callback function signature.
+
+>❗️ Remember to fund your contract with LINK!
+>
+> Making a GET request will fail unless your deployed contract has enough LINK to pay for it. **Learn how to [Acquire testnet LINK](../acquire-link/) and [Fund your contract](../fund-your-contract/)**.
+> 
+
+<div class="remix-callout">
+    <a href="https://remix.ethereum.org/#optimize=false&evmVersion=null&gist=10ae2499f48f2932b38f34a96b124443&runs=200&version=soljson-v0.8.6+commit.11564f7e.js" target="_blank" class="cl-button--ghost solidity-tracked">Deploy this contract using Remix ↗</a>
+    <a href="../deploy-your-first-contract/" title="">What is Remix?</a>
+</div>
+
+```javascript Kovan
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+
+/**
+ * @notice DO NOT USE THIS CODE IN PRODUCTION. This is an example contract. 
+ */
```

Comment:
Maybe add a note after the code sample on how would people use it in production?

Reply:
What do you mean?

Reply:
I mean if we state that people shouldn't use this in the production, then the logical question - what they should use instead, maybe some caveats or whatever else they have to consider when using in the production.

Reply:
It's saying it's just an example contract, and not to use that specific contract in prod. I'll clarify.